client/src/pages/Prospects.js

```diff
@@ -4,6 +4,7 @@ import { makeStyles } from '@material-ui/core/styles';
 import Box from '@material-ui/core/Box';
 import queryString from 'query-string';
 
+import Grid from '@material-ui/core/Grid';
```

Comment:
You're using a grid now instead, is it because of the necessity of having a side panel?

Reply:
Yes, I'm using both Grid and Box for the layout.